xbmc/dialogs/GUIDialogKaiToast.cpp

```diff
@@ -134,6 +135,9 @@ bool CGUIDialogKaiToast::DoWork()
     //  Play the window specific init sound for each notification queued
     SetSound(toast.withSound);
 
+    // Activate haptics for this notification
+    PERIPHERALS::g_peripherals.OnUserNotification();
```

Comment:
where is a better place to place this fn call?

"Add rumble on Kia Toast notification " where is the typo?